docs/integrations/destinations/mssql.md

```diff
@@ -142,6 +143,7 @@ Using this feature requires additional configuration, when creating the source.
 
```

Comment:
Why have 2 changelogs?  They should be the same... Ugh.

Reply:
yeah these changelogs are _wildly_ different. Must have been a different time.... 😓 
![image](https://github.com/airbytehq/airbyte/assets/5741425/021fb009-a789-4748-a08e-9febf9f72719)